roles/operator_bundle_scorecard_tests/files/scorecard-basic-config.yml

```diff
@@ -0,0 +1,21 @@
+kind: Configuration
+apiversion: scorecard.operatorframework.io/v1alpha3
+metadata:
+  name: config
+stages:
+- parallel: true
+  tests:
+  - image: quay.io/operator-framework/scorecard-test:latest
+    entrypoint:
+    - scorecard-test
+    - basic-check-spec
+    labels:
+      suite: basic
+      test: basic-check-spec-test
+  - image: quay.io/operator-framework/scorecard-test:latest
```

Comment:
isnt there any tagged version. 
should we always use the latest ? 
@14rcole

Reply:
We expect to be able to tag the scorecard images to the v1.8.0 version after we upgrade the operator-sdk to that same version as part of CVP-1887.

Reply:
Additionally. this config file is copied from [the operator scorecard container](https://github.com/redhat-operator-ecosystem/operator-scorecard-test-container), which is the container from which scorecard tests are run in DPTP.  Keeping the default configs the same ensures consistency between internal and external test runs.

Reply:
If we want to have the same file as in redhat-operator-ecosystem/operator-scorecard-test-container
than we should document it somewhere. Maybe in the read.me in the same directory.

Reply:
And ideally there shouls he comment at the beginning of file as well.